assets/clickhouse_native_json/Dockerfile

```
@@ -0,0 +1,2 @@
+FROM clickhouse/clickhouse-server:latest
```

Hadolint (2.12.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)